### PR TITLE
[RW-5991][risk=no] Home-view

### PR DESCRIPTION
**Pull request by mhigbyflowers**
This is the landing page for the Data Browser.
Navigation and tooltips will be added in a future PR

### CODE REVIEW

**Review by SrushtiGangireddy — APPROVED** [review]
Looks good to me overall. Thanks.

File: public-ui/src/app/data-browser/views/quick-search/home-view-react.component.tsx [hunk 1]
```diff
@@ -1,22 +1,156 @@
...
+        margin: '0 auto',
+        lineHeight: '2',
+        fontFamily: 'GothamBook, Arial, sans-serif',
+        fontSize: '20px'
+    },
+    dbSubDesc: {
+        padding: '2rem',
+        textAlign: 'center'
+    },
+    iconlinks: {
```
Comment (dolbeew) [inline public-ui/src/app/data-browser/views/quick-search/home-view-react.component.tsx]: Nit: use camel-case to match the other style properties

File: public-ui/src/app/data-browser/views/quick-search/home-view-react.component.tsx [hunk 2]
```diff
@@ -90,78 +230,127 @@ export const dBHomeComponent = (
...
+                            </a>
+                        </div>
+                        <div className='icons' onClick={() => this.iconClickEvent('Intro-Videos')}>
+                            <a>
+                                <img alt='Introductory Videos'
+                                    src='/assets/icons/icons_introductoryvideo.png' /><span className='icon-link'>Introductory Videos</span>
+                            </a>
+                        </div>
+                        <div className='icons' onClick={() => this.iconClickEvent('User-Guide')}>
+                            <a href='../../../assets/pdf/Databrowser_User_Guide_in_RH 5_18_20.pdf' target='_blank' ><img
```
Comment (dolbeew) [inline public-ui/src/app/data-browser/views/quick-search/home-view-react.component.tsx]: Nit: use absolute path to match the other paths:
```suggestion
                            <a href='/assets/pdf/Databrowser_User_Guide_in_RH 5_18_20.pdf' target='_blank' ><img
```

File: public-ui/src/app/data-browser/views/quick-search/home-view-react.component.tsx [hunk 3]
```diff
@@ -90,78 +230,127 @@ export const dBHomeComponent = (
...
+                            <a href={environment.researchAllOfUsUrl + '/frequently-asked-questions/#data-browser-faqs'}>
+                                <img alt='FAQs'
+                                    src='/assets/icons/icons_faqs.png' />
+                                <span className='icon-link'>FAQs</span>
+                            </a>
+                        </div>
+                        <div className='icons' onClick={() => this.iconClickEvent('Intro-Videos')}>
+                            <a>
+                                <img alt='Introductory Videos'
+                                    src='/assets/icons/icons_introductoryvideo.png' /><span className='icon-link'>Introductory Videos</span>
```
Comment (dolbeew) [inline public-ui/src/app/data-browser/views/quick-search/home-view-react.component.tsx]: Nit: move to new line:
```suggestion
                                <img alt='Introductory Videos'
                                    src='/assets/icons/icons_introductoryvideo.png' />
                                <span className='icon-link'>Introductory Videos</span>
```

File: public-ui/src/app/data-browser/views/quick-search/home-view-react.component.tsx [hunk 4]
```diff
@@ -90,78 +230,127 @@ export const dBHomeComponent = (
...
+                            </a>
+                        </div>
+                        <div className='icons' onClick={() => this.iconClickEvent('User-Guide')}>
+                            <a href='../../../assets/pdf/Databrowser_User_Guide_in_RH 5_18_20.pdf' target='_blank' ><img
+                                alt='User Guide' src='/assets/icons/icons_userguide.png' /><span
+                                    className='icon-link'>User Guide</span></a>
```
Comment (dolbeew) [inline public-ui/src/app/data-browser/views/quick-search/home-view-react.component.tsx]: Nit: move closing tag to new line:
```suggestion
                                    className='icon-link'>User Guide</span>
                                </a>
```

File: public-ui/src/app/data-browser/views/quick-search/home-view-react.component.tsx [hunk 5]
```diff
@@ -90,78 +230,127 @@ export const dBHomeComponent = (
...
+                acknowledge the <i>All of Us</i> Research Program in any presentations or publications.<br></br><br></br>
+
+                    <button onClick={() => this.closePopUp()} className='disclaimer-btn'>public data use statement</button>
+                </p>
+                <div style={styles.searchIconLayout}>
+                    <div>
+                        <SearchComponent value={searchWord} onChange={(val) => {
+                            this.handleChange(val);
+                        }}
+                            onClear={() => { this.handleChange(''); }} />
```
Comment (dolbeew) [inline public-ui/src/app/data-browser/views/quick-search/home-view-react.component.tsx]: Call the one-line functions inline, without curly braces to match other instances:
```suggestion
                        <SearchComponent value={searchWord} 
                                                           onChange={(val) => this.handleChange(val)}
                                                           onClear={() => this.handleChange('')} />
```

File: public-ui/src/app/data-browser/views/quick-search/home-view-react.component.tsx [hunk 6]
```diff
@@ -90,78 +230,127 @@ export const dBHomeComponent = (
...
+                <style>{css}</style>
+                <h1 style={{ ...globalStyles.primaryDisplay, ...styles.dBTitle }}>Data Browser</h1>
+                <p style={{ ...styles.dBDesc, ...globalStyles.bodyLead }}>
+                    The Data Browser provides interactive views of the publicly available<i>All of Us </i>
+                    Research Program participant data. Currently, participant provided information, including surveys and physical
+                    measurements taken at the time of participant enrollment, as well as electronic health record data (EHR) are available.
+                EHR data are reported by health care providers and are not participant reported. The <i>All of Us </i>
+                    Research Program data will include more data types over time.<br></br><br></br>
+                    In order to protect participant privacy, the data are de-identified, limited to aggregate counts rounded up to counts of
+                20, and summary demographic information. For more information, please visit our FAQ page.<br></br><br></br>
```
Comment (dolbeew) [inline public-ui/src/app/data-browser/views/quick-search/home-view-react.component.tsx]: Nit: use self-closing tag:
<br/><br/>
```suggestion
                20, and summary demographic information. For more information, please visit our FAQ page.<br/><br/>
```

File: public-ui/src/app/data-browser/views/quick-search/home-view-react.component.tsx [hunk 7]
```diff
@@ -90,78 +230,127 @@ export const dBHomeComponent = (
...
+            const { domainInfo, physicalMeasurementsInfo, surveyInfo, searchWord, popUp } = this.state;
+            return <React.Fragment>
+                <style>{css}</style>
+                <h1 style={{ ...globalStyles.primaryDisplay, ...styles.dBTitle }}>Data Browser</h1>
+                <p style={{ ...styles.dBDesc, ...globalStyles.bodyLead }}>
+                    The Data Browser provides interactive views of the publicly available<i>All of Us </i>
+                    Research Program participant data. Currently, participant provided information, including surveys and physical
+                    measurements taken at the time of participant enrollment, as well as electronic health record data (EHR) are available.
+                EHR data are reported by health care providers and are not participant reported. The <i>All of Us </i>
+                    Research Program data will include more data types over time.<br></br><br></br>
```
Comment (dolbeew) [inline public-ui/src/app/data-browser/views/quick-search/home-view-react.component.tsx]: Nit: use self-closing tag:
```suggestion
                    Research Program data will include more data types over time.<br/><br/>
```

File: public-ui/src/app/data-browser/views/quick-search/home-view-react.component.tsx [hunk 8]
```diff
@@ -90,78 +230,127 @@ export const dBHomeComponent = (
...
+                    In order to protect participant privacy, the data are de-identified, limited to aggregate counts rounded up to counts of
+                20, and summary demographic information. For more information, please visit our FAQ page.<br></br><br></br>
+                    Please read the public data use statement available below for additional information about our unique dataset and how to
+                acknowledge the <i>All of Us</i> Research Program in any presentations or publications.<br></br><br></br>
```
Comment (dolbeew) [inline public-ui/src/app/data-browser/views/quick-search/home-view-react.component.tsx]:
```suggestion
                acknowledge the <i>All of Us</i> Research Program in any presentations or publications.<br/><br/>
```

File: public-ui/src/app/data-browser/views/quick-search/home-view-react.component.tsx [hunk 9]
```diff
@@ -90,78 +230,127 @@ export const dBHomeComponent = (
...
+                    <div style={styles.iconlinks}>
+                        <div className='icons' onClick={() => this.iconClickEvent('FAQ')}>
+                            <a href={environment.researchAllOfUsUrl + '/frequently-asked-questions/#data-browser-faqs'}>
+                                <img alt='FAQs'
+                                    src='/assets/icons/icons_faqs.png' />
+                                <span className='icon-link'>FAQs</span>
+                            </a>
+                        </div>
+                        <div className='icons' onClick={() => this.iconClickEvent('Intro-Videos')}>
+                            <a>
```
Comment (dolbeew) [inline public-ui/src/app/data-browser/views/quick-search/home-view-react.component.tsx:288]: Should be able to use the new `navigate` function switch to intro vids route

File: public-ui/src/app/data-browser/views/quick-search/home-view-react.component.tsx [hunk 10]
```diff
@@ -90,78 +230,127 @@ export const dBHomeComponent = (
...
+                            <a href='../../../assets/pdf/Databrowser_User_Guide_in_RH 5_18_20.pdf' target='_blank' ><img
+                                alt='User Guide' src='/assets/icons/icons_userguide.png' /><span
+                                    className='icon-link'>User Guide</span></a>
+                        </div>
+                    </div>
                 </div>
-            </section></React.Fragment>;
+
+                <section style={styles.results}>
+                    <h5 style={{ ...globalStyles.secondaryDisplay, ...styles.resultHeading }}>
```
Comment (dolbeew) [inline public-ui/src/app/data-browser/views/quick-search/home-view-react.component.tsx:304]: Should also use the `navigate` function for all domain cards

**Pull request merged**